core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -754,4 +758,8 @@ <h3>Canonical Stories</h3>
       right: 43px;
     }
   }
+  :disabled, .add-story-btn:disabled, .disabled,
+  .add-subtopic-btn:disabled,.add-skill-btn:disabled {
+    cursor: not-allowed;
+  }
```

Comment:
Can you add a screenshot of before/after your changes to see how the cursor looks?

Reply:
I have added the video in the "proof that changes are correct" section. PTAL.